docs/getting-started/steps-to-build-deploy-contract.md

```diff
@@ -0,0 +1,80 @@
+- **Inputs and outputs**: What inputs and outputs will your contract require? Inputs could be choices, deposits of funds, or data from an oracle. Outputs could be transactions that result in funds being sent to specified participant addresses. 
+- **Authorizations**: What sort of authorizations would be required for each contract participant? For example, a public key authorization or a role token authorization. 
+- **User experience**: What sort of user experience do you want your contract participants to have? Consider whether or not you want to bring a layer of web development into your contract. For example, you could take advantage of functionalities offered by the Marlowe TypeScript SDK to integrate smart contract capabilities into web pages. 
+
+### Development
+
+As you prepare to start creating the code for your smart contract, take into account what skills you have or the skills of your team members if it is a collaborative effort. 
+
+The Marlowe platform provides the [**Playground**](https://play.marlowe.iohk.io/) where you can use the visual approach with Blockly to drag and drop contract components. Building your contract with Blockly frees you from writing any code. Or you could write code in the Playground using JavaScript, Haskell or the Marlowe DSL. 
+
+Please see these resources for more detailed information about working with the Playground: 
+
+* [**Developer tools > Marlowe Playground**](https://docs.marlowe.iohk.io/docs/developer-tools/playground)
+* [**Tutorials > Overview of the Marlowe Playground**](https://docs.marlowe.iohk.io/tutorials/concepts/playground-overview)
+
+### Simulation and testing
+
+Devote time and effort towards testing your contract in the Playground. Simulation and testing are a critical part of your smart contract development. Simulating your contract allows you to test various possible pathways and scenarios to make sure that your contract behaves as expected. 
+
+The simulation will help to reveal any errors in your contract such as failing to make a payment. You can also verify that a refund occurs if certain conditions are met. If you do not include a timeout, simulating the contract will help to reveal this. For example, if a participant fails to make a deposit, the contract should include a timeout. 
+
+#### To simulate your contract
+
+1. With your contract open in the Playground, select ‘Send to Simulator’ in the upper right area. A panel opens where you can specify deadline parameters and ada amounts. 
+2. Step through the contract’s execution and test various choices, examining different potential pathways. This will give you insight into how your contract will behave under different conditions. 
+
+## Deploying a smart contract
+
+Use Marlowe Runner to deploy your smart contract to the blockchain. Marlowe Runner streamlines and simiplifies the process of deploying smart contracts on Cardano. It eliminates the need for low-level tools and deploying your own backend Runtime instance. 
```

Comment:
Please add a link to Marlowe Runner

Reply:
I'll add these two links below the paragraph: 

* [**Access Runner on the preview network**](https://preview.runner.marlowe.iohk.io/)
* [**Access Runner on the pre-production network**](https://preprod.runner.marlowe.iohk.io/)